Lib/glyphsLib/classes.py

```diff
@@ -2559,15 +2559,17 @@ def bezierMinMax(self, x0, y0, x1, y1, x2, y2, x3, y3):
 
 class GSComponent(GSBase):
     def _serialize_to_plist(self, writer):
+        # NOTE: The fields should come in alphabetical order.
```

Comment:
these comments can be ignored and quickly become obsolete.. Ideally we'd collect various parameters, sort alphabetically and emit them all in the desired order so that the ordering is enforced.

Reply:
I see you +1, so are you going to do anything about this?

Reply:
I was thinking the same that one has to accumulate what is emitted and re-emit in the right order regardless of the order used in the serialize function. So I like the idea. But no, I was not planning to do anything within this PR. It can be addressed separately.

Reply:
fair enough